client/actions/group.js

```diff
@@ -0,0 +1,79 @@
+import axios from 'axios';
+
+/**
+ * @class GroupActions
+ */
+export default class GroupActions {
+  /**
+  * Request to the API to create a group
+  *
+  * @static
+  * @param {String} groupName The name of the group to be created
+  * @param {String} description The description of the group to be created
+  * @param {Array} groupMembers The members of the group to be created
+  * @param {String} token JWToken to access the endpoint
+  * @returns {Object} dispatch object
+  * @memberof GroupActions
+  */
+  static createGroup(groupName, description, groupMembers, token) {
+    return (dispatch) => {
+      dispatch({ type: 'CREATE_GROUP_BEGINS' });
+      return axios.post('https://postit-api.herokuapp.com/api/group', {
```

Comment:
Why are you using the Heroku hosted api directly?
You can use `/api/group`, it will work on both Heroku and localhost.